secure_message/common/utilities.py

```diff
@@ -148,6 +148,9 @@ def add_business_details(messages):
 
 def add_users_and_business_details(messages):
     """Add both user and business details to messages based on data from party service"""
+    if not messages:
```

Comment:
Really small change, but its better standard to have `if messages` then the `else` statement is the `if not`:
```
if messages:
	messages = add_to_details(messages)
    messages = add_from_details(messages)
    logger.info("Successfully added to and from details")
    messages = add_business_details(messages)
    logger.info("Successfully added business details")
    return messages
return messages
```

Reply:
Also is this tested anywhere?

Reply:
I can add a test no problem 👍 I always thought it was better to do ```if not``` but I'm up for changing it

Reply:
I don't think we should do it like this. We should enforce that `add_users_and_business_details` is passed messages. The only place this method is called when there may be no messages is in the `ThreadList` resource in threads.py. We should check there if messages is `None` before attempting to call `add_users_and_business_details`. 

This method should have a check for when `messages` is `None` and raise an exception, something like:

```python
def add_users_and_business_details(messages):
	if messages is None:
		raise ValueError('messages is required')
	
	... existing method ...
```

Reply:
I don't think we should do it like this. We should raise an exception here if no messages are passed, something like:
```python
def add_users_and_business_details(messages):
	if messages is None:
		raise ValueError('messages is required')

	... existing method ...
```

The only place `add_users_and_business_details` is called when messages may be `None` is from the `ThreadList` resource in threads.py. We should deal with it there and check we have messages before calling `add_users_and_business_details`. The other places where the method is called is in the context of a conversation, where there will always be at least one message.